### ISSUE TITLE
OmniFocus 3 not syncing

### ISSUE THREAD

**Issue by jmbrysk** [lra/mackup#1458]
Hello,

I'm new to the tool and recently did the initial setup. While a handful of applications seemed to be automatically detected and synced, OmniFocus 3 was not. It is installed on both Macs.

What can I do to troubleshoot?

**Comment by jmbrysk** [comment]
I changed the title - this appears to be more than just an Omnifocus issue. In fact, only one or two applications seem to be syncing.

I tried explicitly including Omnifocus and other apps in a config file, but it still isn't picked up. What else can I try?

**Comment by icopp** [comment]
Are you using the 10.15 beta? That has lots of problems because of more aggressive sandboxing of everything that touches iCloud Drive or Documents (see #1386 and #1423).

**Comment by jmbrysk** [comment]
Nope, 10.14.6. I did also try switching storage providers from Dropbox to iCloud and back to Dropbox during my troubleshooting.

**Comment by dreed47** [comment]
I am having the same issue and also on 10.14.6.  Many apps are not syncing and I've tried both iCloud and Google Drive.  They just are not picked up, I don't see any error messages or anything.

**Comment by oa-** [comment]
OmniGroup files are located in the ~/Library/Containers folder, e.g. /Users/USERNAME/Library/Containers/com.omnigroup.OmniFocus3 for OmniFocus 3.

Running 'mackup -v' reveals that it's looking in the wrong path:
`  /Users/USERNAME/Library/Preferences/com.omnigroup.OmniFocus.plist
  does not exist
`
